admin/cluster_containers_setup.py

```diff
@@ -0,0 +1,324 @@
+    :ivar user_cert: ``FilePath`` of the user certificate.
+    :ivar user_key: ``FilePath`` of the user key.
+    :ivar client: ``FlockerClient`` conected to the cluster.
+    :ivar reactor: ``Reactor`` used by the client.
+    :ivar nodes: list of `Node` returned by client.list_nodes
+    """
+    def __init__(self, reactor, env, options):
+        """
+        ``ClusterContainerDeployment`` constructor.
+
+        :param reactor: ``Reactor`` we are using.
+        :param env: ``environ`` with the current environment.
+            NOTE: alternative of making it work with env variables pending
+            implementation
+        :param options: ``ContainerOptions`` with the options passed to the
+            the script.
+        """
+        # XXX add the capability to use env variables if the options are not
+        # passed??
+        # self.control_node_address = env['FLOCKER_ACCEPTANCE_CONTROL_NODE']
+        # self.certificates_path = FilePath(
+        #    env['FLOCKER_ACCEPTANCE_API_CERTIFICATES_PATH'])
+        self.options = options
+        try:
+            self.application_template = self.options['template']
+            self.per_node = self.options['apps-per-node']
+            self.control_node_address = self.options['control-node']
+            self.timeout = self.options['wait']
+            if self.timeout is None:
+                self.timeout = 100
+        except Exception as e:
+            sys.stderr.write("%s: %s\n" % ("Missing or wrong arguments", e))
+            sys.stderr.write(e.args[0])
+            sys.stderr.write('\n\n')
+            sys.stderr.write(options.getSynopsis())
+            sys.stderr.write('\n')
+            sys.stderr.write(options.getUsage())
+            raise SystemExit(1)
+
+        certificates_path = FilePath(self.options['cert-directory'])
+        self.cluster_cert = certificates_path.child(b"cluster.crt")
+        self.user_cert = certificates_path.child(b"user.crt")
+        self.user_key = certificates_path.child(b"user.key")
+        self.client = None
+        self.reactor = reactor
+        self.nodes = []
+        self._initialise_client()
+
+    def _initialise_client(self):
+        """
+        Initialise flocker client.
+        """
+        self.client = FlockerClient(
+            self.reactor,
+            self.control_node_address,
+            REST_API_PORT,
+            self.cluster_cert,
+            self.user_cert,
+            self.user_key
+        )
+
+    def _set_nodes(self, nodes):
+        """
+        Set the list of the nodes in the cluster.
+        """
+        self.nodes = nodes
+
+    def deploy(self):
+        """
+        Deploy the new configuration: create the requested containers
+        and dataset in the cluster nodes.
+
+        :return Deferred: that will fire once the request to create all
+            the containers and datasets has been sent.
+        """
+        print "Listing current nodes"
```

Comment:
You should replace explicit `print` statements with either eliot logging or with a class attribute (provided to `__init__`) that can be called with a string.